### PR TITLE
Creates a first website version

### PR DESCRIPTION
**Pull request by djcouto**
This is a first version of the website with the project structure.

### CODE REVIEW

File: data/members/adelinocosta.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (martinhoaragao) [inline data/members/adelinocosta.json:1]: Everyone has their own file? Seems a bit excessive imo. But alright.

File: data/members/franciscoribeiro.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "Name": "Francisco Ribeiro",
+  "Job":"Programmer",
+  "Photo": "https://github.com/FranciscoRibeiro.png"
+}
```
Comment (martinhoaragao) [inline data/members/franciscoribeiro.json]: Those lack of newlines, someone's editor is badly configured 😂

File: themes/kairos-theme/LICENSE.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 YOUR_NAME_HERE
+Copyright (c) 2017 Kairos Website Theme
```
Comment (martinhoaragao) [inline themes/kairos-theme/LICENSE.md]: Any way to make this dynamic? I guess it's too much trouble, but we shouldn't forget to change this later I guess.

File: themes/kairos-theme/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Kairos Website Theme
+
+The Kairos website theme is a one-page-portfolio based on the self-named template made by [Justice](https://freehtml5.co/demos/justice/). Noteworthy features of this Hugo theme are a custom about section, a team section and a contact form.
+
+## Getting Started
+
+Inside the theme folder run:
```
Comment (martinhoaragao) [inline themes/kairos-theme/README.md]: Oi? There's no hugo involved in your getting started?

File: themes/kairos-theme/gulpfile.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+var elixir = require('laravel-elixir');
```
Comment (martinhoaragao) [inline themes/kairos-theme/gulpfile.js:1]: I'm confused by the laravel elixir along with go on hugo. Please explain

**Review by martinhoaragao — CHANGES_REQUESTED** [review]

File: data/members/adelinocosta.json (hunk 1 above)
Comment (djcouto) [reply data/members/adelinocosta.json:1]: This way you can add/remove members easily, for example the two translations members from ILHC.

File: data/members/franciscoribeiro.json (hunk 2 above)
Comment (djcouto) [reply data/members/franciscoribeiro.json]: Ok I will fix it in the next commit :smile:

File: data/members/adelinocosta.json (hunk 1 above)
Comment (martinhoaragao) [reply data/members/adelinocosta.json:1]: I guess it's mostly a matter of personal opinion. But your approach might indeed be best in this case.
Cool 😄

File: themes/kairos-theme/gulpfile.js (hunk 5 above)
Comment (djcouto) [reply themes/kairos-theme/gulpfile.js:1]: https://laravel.com/docs/5.5/mix
I used laravel elixir as a package to help define my asset pipeline. This way I can a several *javascript* and *css/sass* files and compile to a single _.js_ and _.css_. Its easy to use and very helpful as you can see.

File: themes/kairos-theme/README.md (hunk 4 above)
Comment (dinocosta) [reply themes/kairos-theme/README.md]: I also don't happen to know what `snap` is so maybe I would recommend adding a line saying something like: "Make sure you have snap installed" and a link to instructions on how to install snap :)

**Review by dinocosta — APPROVED** [review]
I am going to approve as I've said to @djcouto I would but I would like us to get rid of the Loren Ipsum as soon as possible. Let's keep it up 💪

File: themes/kairos-theme/README.md (hunk 4 above)
Comment (djcouto) [reply themes/kairos-theme/README.md]: :+1:

**Review by martinhoaragao — APPROVED** [review]

**Pull request merged**